src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -333,6 +333,21 @@ void ApplicationManagerImpl::OnApplicationRegistered(ApplicationSharedPtr app) {
   sync_primitives::AutoLock lock(applications_list_lock_);
   const mobile_apis::HMILevel::eType default_level = GetDefaultHmiLevel(app);
   state_ctrl_.OnApplicationRegistered(app, default_level);
+
+  // TODO(AOleynik): Is neccessary to be able to know that registration process
+  // has been completed and default HMI level is set, otherwise policy will
+  // block all the requests/notifications to mobile
```

Comment:
please check this comment. if task closed, remove comment.
And maybe we haven`t any reason to duplicate this comment in 2 repo.